include/circt/Dialect/Arc/ArcPasses.td

```diff
@@ -58,6 +58,24 @@ def Dedup : Pass<"arc-dedup", "mlir::ModuleOp"> {
   ];
 }
 
+def DummyAnalysisTester : Pass<"arc-dummy-analysis-tester", "mlir::ModuleOp"> {
+  let summary = "A dymmy pass to test analysis passes";
+  let constructor = "circt::arc::createDummyAnalysisTesterPass()";
+  let dependentDialects = ["arc::ArcDialect"];
```

Comment:
Can you call this something like `PrintCostModel`, `arc-print-cost-model`? Dummy analysis tester is very generic.

Reply:
you're right, what a dummy name!